### PR TITLE
Added "Good Spirits".

### PR DESCRIPTION
**Pull request by archagon**
1. [x] Project URL: https://github.com/archagon/good-spirits
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -17039,6 +17039,25 @@
...
       "suggested_by": "@vpeschenkov"
+    },
+    {
+      "title": "Good Spirits",
+      "category-ids": [
+        "health"
+      ],
+      "description": "A drink tracker with weekly limits, Untappd support, and HealthKit sync.",
+      "source": "https://github.com/archagon/good-spirits",
+      "itunes": "https://itunes.apple.com/us/app/good-spirits/id1434237439?mt=8&ref=oslist",
```
Comment (vpeschenkov) [inline contents.json]: @archagon Could you replace this link by https://itunes.apple.com/app/good-spirits/id1434237439?

File: contents.json [hunk 2]
```diff
@@ -17039,6 +17039,25 @@
       ],
       "date_added": "Sep 3 2018",
       "suggested_by": "@vpeschenkov"
+    },
+    {
+      "title": "Good Spirits",
+      "category-ids": [
+        "health"
+      ],
+      "description": "A drink tracker with weekly limits, Untappd support, and HealthKit sync.",
```
Comment (vpeschenkov) [inline contents.json]: @archagon Could you remove the dot at the end of the sentence?

File: contents.json [hunk 3]
```diff
@@ -17039,6 +17039,25 @@
...
+      "source": "https://github.com/archagon/good-spirits",
```
Comment (vpeschenkov) [inline contents.json]: @archagon Could you add the license of this source, like this `"license": "gpl-3.0"`

**Comment by vpeschenkov** [comment]
Hello @archagon, thank you for contributing, could you fix the review issues?

**Comment by archagon** [comment]
Done, how about now?

**Comment by vpeschenkov** [comment]
@archagon everything is fine, thank you! 

> I don't think I saw the license tag in the template, that should probably be updated.

Yes, I either. Probably we should fix that 🙂

**Pull request merged**